Tensile/TensileCreateLibrary.py

```diff
@@ -1291,6 +1291,10 @@ def splitExtraParameters(par):
   for logicFile in logicFiles:
     print1("#   %s" % logicFile)
 
+  # Skip experimental libraries (if exists) when building without lazy loading
+  if not globalParameters["LazyLibraryLoading"]:
```

Comment:
Shouldn't this be above the print1 at line 1290?
Otherwise, the names are printed but not included.

Reply:
Yes. That makes sense.